src/components/DevtoolsPane/index.tsx

```diff
@@ -211,7 +211,8 @@ const RowSignalingOptions: React.FC = () => {
 const RowAdvancedOptions: React.FC = () => {
   const [collapsed, setCollapsed] = useState(true);
   const audioStreamingLanguageCode = useAppSelector((state) => state.audioStreamingLanguageCode);
-  const enabledOptions = [audioStreamingLanguageCode !== ""].some((e) => e);
+  const lyraParamsBitrateForm = useAppSelector((state) => state.lyraParamsBitrate);
```

Comment:
変数名の末尾に "Form" は不要だと思います。
```suggestion
  const lyraParamsBitrate = useAppSelector((state) => state.lyraParamsBitrate);
```

Reply:
確認ありがとうございます。ご指摘のとおりです。修正しました。